force-app/main/default/staticresources/pmm/listViewSelectorOverrides.css

```diff
@@ -0,0 +1,10 @@
+.listViewSelector button,
```

Comment:
curious, why a separate file?

Reply:
Doesn't appear to be necessary, moving it. Thanks for the catch!

Reply:
Actually since they are overriding lwc base components it does need to be an override.

Reply:
but you pulled them out of the other static resource file? can't we keep them all in one file?

Reply:
future-proofing. got it.